app/MindWork AI Studio/Components/ChatComponent.razor.cs

```diff
@@ -320,6 +325,24 @@ private async Task ProfileWasChanged(Profile profile)
         
         await this.ChatThreadChanged.InvokeAsync(this.ChatThread);
     }
+    
+    private async Task ChatTemplateWasChanged(ChatTemplate chatTemplate)
+    {
+        this.currentChatTemplate = chatTemplate;
+        this.disableProfile = !chatTemplate.AllowProfileUsage;
```

Comment:
Not required, right? The selection of profiles is controlled by `Disabled`, correct?